environment.yml

```diff
@@ -62,7 +62,6 @@ dependencies:
   # downstream packages
   - dask-core<=2024.2.1
```

Comment:
```suggestion
  - dask-core
```

Reply:
@mroeschke Thanks for your instructions. This reveals that I didn't understand version pinning, but I learned something.